Games/types/Mafia/roles/cards/CurseVote.js

```diff
@@ -0,0 +1,34 @@
+const Card = require("../../Card");
+const { PRIORITY_KILL_DEFAULT } = require("../../const/Priority");
+
+module.exports = class CurseVote extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Victim": {
+                states: ["Night"],
+                flags: ["voting"],
+                action: {
+                    priority: PRIORITY_KILL_DEFAULT - 1,
+                    run: function () {
+                        this.actor.role.data.victim = this.target;
+                    }
+                }
+            },
+            "Target": {
+                states: ["Night"],
+                flags: ["voting"],
```

Comment:
Also, specifying no targets here makes it so that only the diabolist cannot be selected as the target. What is the intended behavior here? Can you select mafia & diabolist as the targets, or should they be excluded?
From what I remember, it should be able to select any target, no?

Reply:
I don't think Diab could select themselves on EM.

Reply:
I think it is better to explicitly specify the target even if using the default one 

i.e. includes all, excludes self